prow/jobs/kyma/components/compass-runtime-agent/compass-runtime-agent-tests.yaml

```diff
@@ -21,6 +21,7 @@ presubmits: # runs on PRs
       path_alias: github.com/kyma-project/kyma
       cluster: untrusted-workload
       max_concurrency: 10
+      optional: true
```

Comment:
We could consider narrowing the scope (run_if_changed option) of the pipeline from the entire installation directory, to subfolders with our and dependent components.